common/tests/threadpool_int/threadpool_int.c

```diff
@@ -842,7 +843,7 @@ TEST_FUNCTION(timer_cancel_restart_works_runs_periodically)
     THANDLE(THREADPOOL_TIMER) timer = threadpool_timer_start(threadpool, 100, 500, work_function, (void*)&g_call_count);
     ASSERT_IS_NOT_NULL(timer);
 
-    // Timer should run 4 times in about 2.1 seconds
+    // Timer should run 4 times in about 3 seconds
     ASSERT_ARE_EQUAL(INTERLOCKED_HL_RESULT, INTERLOCKED_HL_OK, InterlockedHL_WaitForValue64(&g_call_count, 4, 3000));
     LogInfo("Timer completed 4 times");
```

Comment:
I have to say I don't like this kind of test.
It is brittle. Something will delay the system at some point and lo and behold, next we have someone debugging a flakiness.

Let's create a task under things that never get done for fixing it...

Reply:
created the task : [https://msazure.visualstudio.com/One/_workitems/edit/31672710](https://msazure.visualstudio.com/One/_workitems/edit/31672710)